### PR TITLE
Updated two quick starts and moved existing two to an archive folder

### PR DESCRIPTION
**Pull request by dkershaw10**
to an archive folder

### CODE REVIEW

File: quickstart-templates/create-fic-for-github-actions/main.bicep [hunk 1]
```diff
@@ -1,40 +1,50 @@
...
 
-// The service principal needs to be assigned the necessary role to access the resources
-// In this example, it is assigned with the `Contributor` role to the resource group
-// which will allow GitHub actions to access Azure resources in the resource group via Az PS/CLI
-var roleAssignmentName = guid('githubActions', roleDefinitionId, resourceGroup().id)
-resource roleAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+var roleAssignmentName = guid(resourceGroup().id, 'githubactions', contributorRoleDefinitionId)
+
+// Assign the group as a storage blob reader role
+resource storageBlobReadersRoleAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
```
Comment (jason-dou) [inline quickstart-templates/create-fic-for-github-actions/main.bicep:41]: The resource symbolic name should be `contributorRoleAssignment` or similar. The comment needs to be updated as well.

File: quickstart-templates/create-fic-for-github-actions/README.md [hunk 2]
```diff
@@ -1,23 +1,45 @@
...
+The following video demonstrates the an end-to-end scenario of configuring workload identity federation and a GitHub Action workflow, so that the GitHub Action can log into Microsoft Entra, build and deploy a web app into an Azure App Service. This is all accomplished **without the use of any secrets**.
 
-> **Note2**: This template sample **only** configures the Microsoft Entra ID portion (to enable workload identity federation). Additional configuration steps are also required on the GitHub side, to ensure that the federation works end-to-end. See [Use GitHub Actions to connect to Azure](https://learn.microsoft.com/azure/developer/github/connect-from-azure?tabs=azure-cli%2Cwindows#use-the-azure-login-action-with-openid-connect), but skip the sections on "Create a Microsoft Entra application and service principal" and "Add federated credentials", as the following Bicep template replaces those sections.
+![GitHub Action deploys web app to Azure App Services using federated identity credentials](../../docs/media/github-actions-and-workload-identities.mp4)
```
[the remark this answers is not in the document]
Comment (dkershaw10) [reply quickstart-templates/create-fic-for-github-actions/README.md:5]: Yeah - I'm also trying to figure this out...

File: quickstart-templates/create-fic-for-github-actions/main.bicep (hunk 1 above)
Comment (dkershaw10) [reply quickstart-templates/create-fic-for-github-actions/main.bicep:41]: Good catch - on your code sample :)

File: quickstart-templates/create-fic-for-github-actions/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dkershaw10) [reply quickstart-templates/create-fic-for-github-actions/README.md]: OK figured it out - the videos are now embedded.

**Pull request merged**